.github/workflows/ci.yml

```
@@ -279,6 +279,7 @@ jobs:
 
       - name: Check dev drive space
         working-directory: ${{ env.ZED_WORKSPACE }}
+        # `setup-dev-driver.ps1` creates a 100GB drive, with CI taking up ~45GB of the drive.
         run: ./script/exit-ci-if-dev-drive-is-full.ps1 55
```

```suggestion
        run: ./script/exit-ci-if-dev-drive-is-full.ps1 65
```
The number here sets the size for tests.